emva1288/unittests/test_results.py

```diff
@@ -300,3 +302,68 @@ def test_nans(self):
         self.assertIs(r.DSNU1288_DN(), np.nan)
 
         del r
+
+
+class TestRoutines(unittest.TestCase):
+
+    # instantiate a bayer filter and a wrong filter
+    bayer_filter = get_bayer_filter(0, 1, 1, 0, 1000, 1000)
+    wrong_filter = np.tile([[1, 1, 1, 0], [1, 0, 0, 1]], (500, 250))
+
+    # intantiate masked images based in the masks
+    img = np.abs(np.random.random([1000, 1000]))
+    wrong = np.ma.array(img, mask=wrong_filter)
+    masked = np.ma.array(img, mask=bayer_filter)
+
+    # save the filtered images, masked and with no mask
+    res = high_pass_filter(img, 5)
+    img_filt = res['img']
+    im_num = res['multiplicator']
```

Comment:
as far as I can see, you are using img_filt/img_num everywhere, so just do it here and forget the 'multiplicator'
I mean define `img_filtered = res['img']/res['multiplicator']` instead of the separated elements